content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+---
+title: Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+excerpt: A collection of articles to help you get started with your new DNSimple account quickly.
+categories:
+- DNSimple
+---
+
+# Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Welcome to [DNSimple](/articles/dnsimple-services/)! We are going to make your transition from ClickFunnels as simple as possible. [Simple](/articles/how-is-dnsimple-pronounced/) is in our name, and we're not going to let you down.
+
+## Step 1: Setting up a DNSimple account and trial
+
+<div class="section-steps" markdown="1">
+1. Go to [DNSimple.com](https://dnsimple.com). In the upper right corner of the screen, click the button that says <label>Get Started</label>.
+1. Complete the form using the **same email address that is associated with your ClickFunnels account**. Enter a password and confirm it. Click <label>Join DNSimple</label>.
+1. Now choose a plan. Select <label>Continue</label> under <label>Individual</label>, then choose <label>Get Started</label> under <label>Personal</label> or <label>Professional</label>, depending on your needs.
```

Comment:
Did we miss the "Individual" slide to set up the account?

Reply:
@SarahCaminiti1 can you please clarify?

Reply:
When you first set up your account you're asked to select if you're an individual or a business, I've seen a handful of CF folks see this and since they're a "business" select that and then panic that the cheapest plan is Professional

Reply:
Clicking the individual slide is listed here in these instructions. Please lemme know if I'm missing something. 😅